### PR TITLE
Add blind signature and refund flowcharts

### PR DESCRIPTION
**Pull request by twzjwang**

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -228,27 +228,34 @@ \subsubsection{Components}
...
 
-RSA blind signature\cite{cryptoNote} is used in our research. The data provider chooses a random number $r$ as a blinding factor. $e$ is the broker's public key. To prevent the message $m$ from being known by the broker, the data provider sends blinding message $c$ to the broker instead of $m$.
+\begin{figure}[h]
+	\centering
+	\includegraphics[width=3.3 in]{blind_signature}
+	\caption{The scheme of blind Signature.}
```
Comment (jkrvivian) [inline paper.tex]: signature should be lower case.

File: paper.tex [hunk 2]
```diff
@@ -301,7 +308,17 @@ \subsubsection{Trading}
...
+
+It is probable that the streaming data sources are delayed or even interrupted after the consumers pay the subscription fee. To protect consumer rights, the subscription fee are not transferred to the data provider until data is generated and published to the MAM channel. If the expected data is not available, consumers can request refunds. We assume that a very small percentage of consumers in the product contract are irrational and/or malicious. Each consumer can vote for a refund at any time. If the ratio of consent votes of refunding is higher than the $threshold$ at the $i$th piece of data, the subscription fee is proportionally transferred to the data provider, broker and every consumer. Figure \ref{fig:refund} shows the refund process.
+
+\begin{figure}[h]
+	\centering
+	\includegraphics[width=2 in]{refund}
+	\caption{The process of the refunding.}
```
Comment (jkrvivian) [inline paper.tex]: The process of refund.

File: paper.tex [hunk 3]
```diff
@@ -228,27 +228,34 @@ \subsubsection{Components}
...
+There is inherent risk in revealing session keys to brokers since contents may be copied by brokers, which would result in data providers' losses. Therefore, a blind signature is used to prevent session key copying for such circumstances. Blind signature\cite{blindSig} is a form of digital signature where the message is first "blinded" by a random "blinding factor", then passed to a signer to sign. Figure \ref{fig:blind_signature} illustrates the steps of blind signature. The resulting message, along with the blinding factor, can be later verified with the signer's public key. In our system design, brokers would perform blind signatures during the process of adding new products for data providers, in order to send the secret key of the MAM channel to the smart contract without knowing it.
```
Comment (jserv) [inline paper.tex]: Shall it be "an inherent risk"?

File: paper.tex [hunk 4]
```diff
@@ -228,27 +228,34 @@ \subsubsection{Components}
...
 
-$S$ is the broker's signature of $C$. In RSA system, $ed$ is equal to 1. To remove the blinding factor, the data provider computes the following calculation.
+$S$ is the signer's signature of $C$. In RSA system, $ed$ is equal to 1. To remove the blinding factor, the user computes the following calculation.
 
 \begin{equation}
 \frac{S}{r}= \frac{C^d}{r} = \frac{(r^e m)^d}{r} = \frac{r^{ed} m^d}{r} = m^d
 \end{equation}
  
-The data provider get $m^d$ which is the broker's signature of the message $m$. At the same time, the broker does not know $m$ and the session key is protected. 
+The user get $m^d$ which is the signer's signature of the message $m$. At the same time, the signer does not know $m$ and the session key is protected.
```
Comment (jserv) [inline paper.tex]: s/get/gets/

File: paper.tex [hunk 5]
```diff
@@ -301,7 +308,17 @@ \subsubsection{Trading}
...
+
+It is probable that the streaming data sources are delayed or even interrupted after the consumers pay the subscription fee. To protect consumer rights, the subscription fee are not transferred to the data provider until data is generated and published to the MAM channel. If the expected data is not available, consumers can request refunds. We assume that a very small percentage of consumers in the product contract are irrational and/or malicious. Each consumer can vote for a refund at any time. If the ratio of consent votes of refunding is higher than the $threshold$ at the $i$th piece of data, the subscription fee is proportionally transferred to the data provider, broker and every consumer. Figure \ref{fig:refund} shows the refund process.
```
Comment (jserv) [inline paper.tex]: Shall we explain consent votes?

**Pull request merged**